src/assets/buddy.svg

```diff
@@ -0,0 +1 @@
+<svg id="Capa_1" enable-background="new 0 0 512 512" height="512" viewBox="0 0 512 512" width="512" xmlns="http://www.w3.org/2000/svg"><g><path d="m126 211c-46.869 0-85-38.131-85-85s38.131-85 85-85 85 38.131 85 85-38.131 85-85 85z"/><path d="m386 211c-46.869 0-85-38.131-85-85s38.131-85 85-85 85 38.131 85 85-38.131 85-85 85z"/><g><g><path d="m407 283.5c0-19.453 7.716-37.127 20.23-50.162-12.807 4.938-26.704 7.662-41.23 7.662-34.78 0-66.002-15.522-87.107-40h-84.685c34.507 26.53 56.792 68.208 56.792 115v155h145.001v-80c0-8.284 6.716-15 15-15s15 6.716 15 15v80h39.999c8.284 0 15-6.716 15-15v-100h-21.5c-39.977 0-72.5-32.523-72.5-72.5z"/><path d="m501 355.835v-.022c-.085.007-.079.014 0 .022z"/></g><path d="m497 326h-17.5c-23.435 0-42.5-19.065-42.5-42.5s19.065-42.5 42.5-42.5c17.92 0 32.5 14.58 32.5 32.5v37.5c0 8.284-6.716 15-15 15z"/><g><g><path d="m190.731 221.001c-18.451 12.612-40.744 19.999-64.731 19.999-14.525 0-28.421-2.722-41.227-7.659 12.512 13.034 20.227 30.707 20.227 50.159 0 39.977-32.523 72.5-72.5 72.5h-21.5v100c0 8.284 6.716 15 15 15h40.001v-80c0-8.284 6.716-15 15-15s15 6.716 15 15v80h144.999v-155c0-39.423-19.945-74.271-50.269-94.999z"/><path d="m11 355.813v.032c.115-.011.124-.022 0-.032z"/></g><path d="m0 311v-37.5c0-17.92 14.58-32.5 32.5-32.5 23.435 0 42.5 19.065 42.5 42.5s-19.065 42.5-42.5 42.5h-17.5c-8.284 0-15-6.716-15-15z"/></g></g></g></svg>
```

Comment:
SVG soubory jsou skvělé, ale vzhledem k tomu, že těch ikonek budeme mít určitě víc, by se nám mohla vyplatit nějaká knihovna? Já třeba ráda používám [https://fontawesome.com](https://fontawesome.com) (ten umím na Typescript s Reactem i nastavit) a nebo úplně jednoduše [https://icons.getbootstrap.com](https://icons.getbootstrap.com). 

Ale určitě to budeme řešit kdyžtak v dalším commitu, jen že mě to teď napadlo. :smile:

Reply:
Knihovna mě nenapadla, ale je určitě good point 👍 pokusím se to přenastavit 😉

Reply:
To je asi na další commit, není to tak jednoduché - musíme si vybrat tu knihovnu (asi na callu?), stáhnout ji se správným package managerem a specificky pro React a vypisování icon je ještě o trochu jiné kvůli Typescriptu. :joy: Zrovna jsem se s tím včera prala, tak bych to možná mohla udělat já, když mám řešení živě v paměti.

Reply:
No zní to zajímavě 😅 ok, tak se domluvíme pak na callu. Asi jedno, kdo to bude dělat, naučit se to stejně musíme obě.